Neos.Neos/Classes/Controller/Frontend/NodeController.php

```diff
@@ -187,9 +187,9 @@ public function previewAction(string $node): void
      * @throws \Neos\Flow\Mvc\Routing\Exception\MissingActionNameException
      * @throws \Neos\Flow\Session\Exception\SessionNotStartedException
      * @throws \Neos\Neos\Exception
-     * @Flow\SkipCsrfProtection We need to skip CSRF protection here because this action could be called
-     * with unsafe requests from widgets or plugins that are rendered on the node
-     * - For those the CSRF token is validated on the sub-request, so it is safe to be skipped here
+     * We need to skip CSRF protection here because this action could be called with unsafe requests from widgets or plugins that are rendered on the node - For those the CSRF token is validated on the sub-request, so it is safe to be skipped here
+     * @Flow\SkipCsrfProtection
+     * @Flow\IgnoreValidation("node")
```

Comment:
I don't understand why this should be needed – IIRC `IngoreValidation` should be only needed for object arguments!? What happens if you omit this?

Reply:
If I submit a form which redirects to a node (e.g. via Neos.Form or a Neos.Fusion:FormBuilder rendered form) _without_ this change, the request end in a 404 and I get the error in the logs `CSRF: token was empty but a valid token is required for Neos\Neos\Controller\Frontend\NodeController::showAction()`.

Reply:
That should be solved by the SkipCsrfProtection, the IgnoreValidation is also necessary and gives you the same error otherwise?

Reply:
Yes, exactly @kitsunet. Without the `IgnoreValidation`, I get the mentioned error.

Reply:
i wonder if the problem here is in any way possibly related to https://github.com/neos/neos-development-collection/issues/4909 ? ill have to experiment with this myself

Reply:
@bwaidelich and @kitsunet i guess this is related to our misconception that the NodeController is infact part of the backend authentication?
But then this part should be a 8.3 bugfix?